UploadingFile.ipynb

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
Need a python class... ipynb won't work... and boto3 is 100 steps later... 

If u are saving this for future reference... 

Please create a new branch and commit there... 

Upstream will need to be only working code